doc/release-notes/release-5.sgml

```diff
@@ -122,6 +124,35 @@ Most user-facing changes are reflected in squid.conf (see below).
    the helpers functionality to rebuild it as needed.
 
 
+<sect1>Loop Detection in Content Delivery Networks
+<p>Details in <url url="https://tools.ietf.org/html/rfc8586" name="RFC 8586">
+
+<p>Squid now uses the CDN-Loop header as a source for loop detection.
+    
```

Comment:
> Yeah, but that is the end of the file which is not being touched by this change.

@yadij , it complains about this line (131), which was modified and is not the end of the line.